.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -367,30 +368,30 @@ kernel_matrix_testing_setup_env_x64:
         echo "These jobs do not permit retries. The go tests are retried a user-specified number of times automatically. In order to re-run the tests, you must trigger the pipeline again"
         'false'
       fi
-    - MICRO_VM_IP=$(cat $CI_PROJECT_DIR/stack.output | grep $ARCH-$TAG | cut -d ' ' -f 2)
-    - MICRO_VM_NAME=$(cat $CI_PROJECT_DIR/stack.output | grep $ARCH-$TAG | cut -d ' ' -f 1)
+    - MICRO_VM_IP=$(cat $CI_PROJECT_DIR/stack.output | grep $ARCH-$TAG | grep $TEST_SET | cut -d ' ' -f 2)
```

Comment:
The linkage between `TEST_SET` and the vmsets in `vmconfig-*.json` files feels a bit weak. The `arch` and `tag` are separate explicit fields, but the `TEST_SET` is only a name prefix? I'd like that to be stronger if possible, so it is hard to mess up. If the `grep` could end up as `grep `$ARCH-$TAG-$TEST_SET` that might be best.

Reply:
Ill think about how we can accomplish this.